src/components/scenes/CreateWalletImportScene.tsx

```diff
@@ -46,7 +46,16 @@ const CreateWalletImportComponent = (props: Props) => {
 
   const handleNext = useHandler(async () => {
     textInputRef.current?.blur()
-    const cleanImportText = importText.trim()
+    let cleanImportText = importText.trim()
+
+    // Clean up mnemonic seeds
+    const cleanImportTextArray = cleanImportText.split(' ')
+    if (cleanImportTextArray.length > 1) {
+      cleanImportText = cleanImportTextArray
+        .filter(part => part !== ' ') // remove extra spaces
```

Comment:
can't possibly have an empty ' ' as `part` since that's your split character. But you can have an empty string if there are two spaces in a row. Change to

`.filter(part => part !== '') // remove empty spaces`